### PR TITLE
Fix docs warning 

### PR DESCRIPTION
**Pull request by sunakshi96**
Fixes #

- [ ] Add documentation.
- [ ] Add tests.
- [ ] Add a changelog entry.
- [ ] Add your name in the contributors file.
- [ ] If you changed the HTTP API, update the [API_VERSION](https://github.com/Kinto/kinto/blob/master/kinto/__init__.py#L15) constant and add an API changelog entry [in the docs](https://github.com/Kinto/kinto/blob/master/docs/api/index.rst)

r? @

### CODE REVIEW

File: docs/kinto-admin.rst [hunk 1]
```diff
@@ -0,0 +1,260 @@
+Kinto Web Administration Console
+################################
+
+.. image:: images/kinto-admin-login.png
+    :align: center
+    :width: 800px
+
+
+
+A Web admin UI to manage data from a *Kinto* `<https://kinto.readthedocs.io/>`_ server: *Demo* `<http://kinto.github.io/kinto-admin/>`_.
```
Comment (gabisurita) [inline docs/kinto-admin.rst]: You can probably point to https://kinto.dev.mozaws.net/v1/admin/ instead.

File: docs/kinto-admin.rst [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+  (using fine-grained permissions)
+- Take advantage of
+  :ref:`schema validation <collection-json-schema>`
+  if you need it
+- :ref:`Track changes <api-history>` on your application data
+
+
+.. _enable:
+
+Prerequisites
```
Comment (gabisurita) [inline docs/kinto-admin.rst]: I guess the next sections make more sense if we are developing Kinto Admin. If we are only using the plugin that is shipped with the server, which is already precompiled, I would simply give instructions on how to enable using a config file and a link to kinto admin repository if someone wants to customize or contribute.

File: docs/index.rst [hunk 3]
```diff
@@ -11,6 +11,7 @@ Table of content
    :maxdepth: 1
 
    overview
+   kinto-admin
```
Comment (gabisurita) [inline docs/index.rst]: I think kinto-admin isn't that relevant to be in the second position of the documentation. I would probably put in the tutorial section, IDK.

**Comment by sunakshi96** [comment]
Hey 
I I have just opened the another issue. And deleted the kinto-admin.rst file 
Thanks gabisurita :-)

**Comment by sunakshi96** [comment]
I think we can handle the issue 'Document built-in plugin for kinto-admin" to describe the enabling and features of plugin. 
.....and the  plugin awaits to be rescued  :-)

**Pull request closed without merging**